meow/frontend/src/components/EditPost/index.js

```diff
@@ -149,23 +150,53 @@ class EditPost extends React.Component {
       });
   };
 
-  sendNow = () => {
-    const { postId } = this.props.match.params;
+  popSectionerror = empty => {
+    if (empty == "section") {
+      notification.open({
+        message: "Send Failed :(",
+        description: "No section was selected",
+        icon: <Icon type="close-circle" style={{ color: "#FF0000" }} />
+      });
+    } else if (empty == "url") {
+      notification.open({
+        message: "No url entered :(",
+        description: "Please enter a url",
+        icon: <Icon type="close-circle" style={{ color: "#FF0000" }} />
+      });
+    }
+  };
 
-    this.savePostPromise(postId).then(data => {
-      if (data) {
+  sendNow = data => {
+    const { postId } = this.props.match.params;
+    if (
+      data &&
+      this.state.section &&
+      this.state.story_url !== null &&
+      this.state.story_url !== ""
```

Comment:
Ahh sorry about this, I think in the issue title I said that the url must be there for send now but it actually doesn't have to. So even if there is no url, it should still be able to send now